pkg/operator/controller/names.go

```diff
@@ -1,5 +1,23 @@
+/*
+Copyright 2022.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package nodeobservabilitycontroller
 
 const (
-	AgentName = "node-observability-agent"
+	// AgentServiceName is the name of the service backed up by the agent's daemonset.
+	AgentServiceName = "node-observability-agent"
+	// nolint
```

Comment:
because `golangci-lint` doesn't like when `secret` is a part of the variable name, which is non sense. To not disable the check completely - I excluded only this variable.
Will change the comment to describe this, thanks!